### ISSUE TITLE
translations?

### ISSUE THREAD

**Issue by x4m3** [mssun/passforios#208]
hi,
is it possible to translate the app in other languages? i'd be happy to send a pr :)

i dont know ios development, so i dont know how to implement multiple languages.

thanks

**Comment by mssun** [comment]
I'm happy to work on localization an i8n. I don't know exactly how iOS handle this, but I would like to work on this. I guess we should extract strings and come up a table to different languages.

**Comment by SimplyDanny** [comment]
I have started to work on this one. As this is some work, I just wanted to inform you to avoid duplicated work.

**Comment by mssun** [comment]
@SimplyDanny Thank you so much. Probably, you have to refactor some code to support multi languages.

**Comment by SimplyDanny** [comment]
Yes, I have already noticed that. But so far, I’m making some progress.

**Comment by mssun** [comment]
@SimplyDanny You can put them in separate PRs.

**Comment by x4m3** [comment]
if you need help for translating strings in french, i can help! (and maybe russian too)

**Comment by SimplyDanny** [comment]
I would propose I finish the work for the English part first. The result will contain at least two files with strings which may be translated into other languages. This can be done in separate PRs. Of course you can already start to find appropriate translations for all the strings in the app. 😉

**Comment by SimplyDanny** [comment]
Now that #242 is merged people can work on translations. Perhaps we could use this thread to list who is working on which language.

**Comment by SimplyDanny** [comment]
Myself can do the German. 🇩🇪

**Comment by x4m3** [comment]
i can do the french one, maybe the russian one.--Отправлено из мобильной Яндекс.Почты21.01.2019, 17:49, "Danny Mösch" <notifications@github.com>:Now that #242 is merged people can work on translations. Perhaps we could use this thread to list who is working on which language.
